packages/components/inputs/select-utils/src/create-select-styles.ts

```diff
@@ -10,6 +10,26 @@
 import { ReactNode } from 'react';
 import { designTokens } from '@commercetools-uikit/design-system';
 
+export type THorizontalConstraint =
```

Comment:
Do we actually want to support all horizontal constraints for both `min` and `max` props?
I doesn't seem the `1` value, for instance to be allowed for the `max` prop.

Reply:
@FilPob, maybe we can have a discussion on a range of values we want for then min and max horizontal values.

Reply:
@ddouglasz has this already been addressed?